process/management/commands/file_worker.py

```diff
@@ -27,20 +28,28 @@ def process(self, connection, channel, delivery_tag, body):
             self.logger.debug("Received message %s", input_message)
 
             upgraded_collection_file_id = None
+
+            message = input_message
```

Comment:
Having two names for the same object is confusing (any modifications to `message` will also be made to `input_message` – they refer to the same object in memory). Please update to just use a single variable.

Reply:
Good point in this. However, using two variables, as I dont want to publish modified input_message (ugly and if the code grwos, it can be misleading).